### PR TITLE
Feat/cmc 1856 claimant response

### PR DESCRIPTION
**Pull request by jaberelfasi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CMC-1856
https://tools.hmcts.net/jira/browse/CMC-1982

### Change description ###
- Claimant intention multiparty refactor
- E2E and API tests are updated and it passes locally

Service Repo CR: https://github.com/hmcts/civil-service/pull/645
Camunda Repo CR: https://github.com/hmcts/civil-camunda-bpmn-definition/pull/135

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/systemUpdate-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
```
Comment (bhssamray) [inline ccd-definition/AuthorisationCaseEvent/systemUpdate-nonprod.json:1]: Is this going under the prod-nonprod release plan. Or directly merged to master?

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -907,7 +907,6 @@
       {
         "UserRoles": [
           "caseworker-civil-admin",
```
Comment (bhssamray) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:909]: If this is going with the prod-nonprod release technique, we should make all changes to existing fields in the nonprod file and take out the base one to the prod file (so the base file no longer contains it).

File: ccd-definition/AuthorisationCaseField/systemUpdate.json [hunk 3]
```diff
@@ -149,6 +149,12 @@
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "R"
   },
+  {
```
Comment (bhssamray) [inline ccd-definition/AuthorisationCaseField/systemUpdate.json:152]: Should this not be part of systemUpdate-nonprod.json?

File: ccd-definition/AuthorisationCaseEvent/systemUpdate-nonprod.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    "CaseEventID": "NOTIFY_RESPONDENT_SOLICITOR2_FOR_CLAIMANT_CONFIRMS_TO_PROCEED",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRU"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOTIFY_RESPONDENT_SOLICITOR2_FOR_CLAIMANT_CONFIRMS_NOT_TO_PROCEED",
+    "UserRole": "caseworker-civil-systemupdate",
+    "CRUD": "CRU"
+  }
```
Comment (akriti-hmcts) [inline ccd-definition/AuthorisationCaseEvent/systemUpdate-nonprod.json:13]: add this to base file please as this is going to be released asap

File: ccd-definition/AuthorisationCaseField/staff-nonprod.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    "UserRole": "caseworker-civil-staff",
+    "CRUD": "R"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "applicant2ResponseDate",
+    "UserRole": "caseworker-civil-staff",
+    "CRUD": "R"
+  }
+]
```
Comment (akriti-hmcts) [inline ccd-definition/AuthorisationCaseField/staff-nonprod.json:135]: this one too move to base file please

**Review by akriti-hmcts — COMMENTED** [review]
please move all the changes to base file as the ticket will be released asap

File: ccd-definition/AuthorisationCaseField/systemUpdate.json (hunk 3 above)
Comment (sherlynkhaw) [reply ccd-definition/AuthorisationCaseField/systemUpdate.json:152]: old comment but non-prod files not needed

**Review by sankaviv1 — DISMISSED** [review]

**Review by Gareth40343 — APPROVED** [review]

**Review by HarryH96 — APPROVED** [review]

**Review by akriti-hmcts — APPROVED** [review]

**Pull request closed without merging**